docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +167,250 @@ $menu->reorderChildren(
 );
 ```
 
+## Tabs
+
+Many elements of the back-office interface, such as System Info or Location View, are built using tabs.
+
+You can extend existing tab groups with new tabs, or create your own tab groups.
+
+### Adding a new tab group
+
+To create a custom tab group with additional logic you need to create it at the level of compiling the Symfony container, using a CompilerPass:
+
+``` php
+// src/AppBundle/DependencyInjection/Compiler/CustomTabGroupPass.php
+
+namespace AppBundle\DependencyInjection\Compiler;
+
+use EzSystems\EzPlatformAdminUi\Tab\TabGroup;
+use EzSystems\EzPlatformAdminUi\Tab\TabRegistry;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+class CustomTabGroupPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition(TabRegistry::class)) {
+            return;
+        }
+
+        $tabRegistry = $container->getDefinition(TabRegistry::class);
+        $tabGroupDefinition = new Definition(
+            TabGroup::class,  // or any class that extends TabGroup
+            ['custom-tab-group']
+        );
+        $tabRegistry->addMethodCall('addTabGroup', [$tabGroupDefinition]);
+    }
+}
+```
+
+A shorter way that you can use when no custom logic is required, is to add your own tab with the new group.
+If a tab's group is not defined, it will be created automatically.
+
+A new tab group can be rendered using the `ez_platform_tabs` Twig helper:
+
+``` jinja
+<div class="my-tabs">
+    {{ ez_platform_tabs('custom-tab-group') }}
+</div>
+```
+
+This will render the group and all tabs assigned to it.
+
+### Adding a new tab
+
+Before you add a tab to a group you must create the tab's PHP class and define it as a Symfony service with the `ezplatform.tab` tag:
+
```

Comment:
Could we add a note: "See our doc about using Services"

Reply:
Oh, and where do we have our doc on services?